libcxx/benchmarks/algorithms/set_intersection.bench.cpp

```diff
@@ -0,0 +1,223 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Can you provide benchmark results in your PR description? We generally do that for optimizations.